lib/logstash/filters/mutate.rb

```diff
@@ -288,6 +291,17 @@ def convert_float(value)
     return value.to_f
   end # def convert_float
 
+  def convert_boolean(value)
+    if value == "true"
+      return true
+    elsif value == "false"
+      return false
+    else
+      @logger.warn("Failed to convert #{value} into boolean. Acceptable values are \"true\" and \"false\".")
+      return value
```

Comment:
@jsvd @purbon As I understood it, the `next` there was supposed to cause a "fail" so the add_field would not happen.  Clearly this is not the case, and it probably isn't the case for other failures from the `convert` method.